### DISCUSSION TITLE
PaginationEdges - Support for Icon props

### DISCUSSION THREAD

**Discussion by cchin25** [mantinedev/mantine#4137] [Feature requests]
Hello,

Our application has strict 508 compliance guidelines.  Currently the JAWS screen reader lists the `<Icon/>` component within `PaginationEdges` in the graphics list which is undesired behavior.  We'd like to be able to pass down additional props to the `PaginationEdges` component.

Something as simple as ` <Icon className={classes.icon} size={getIconSize(ctx.stylesApi.size)} {...iconProps} />` should do the trick.

Thank you!

**Comment by rtivital** [comment]
You can do that by providing custom icon components – https://mantine.dev/core/pagination/#change-icons
